test/e2e/virtual/apiexport/virtualworkspace_test.go

```diff
@@ -367,39 +353,96 @@ func TestAPIExportPermissionClaims(t *testing.T) {
+
+	claimedGVRs := []schema.GroupVersionResource{
 		{Version: "v1", Resource: "configmaps"},
 		{Version: "v1", Resource: "secrets"},
 		{Version: "v1", Resource: "serviceaccounts"},
 		{Group: "rbac.authorization.k8s.io", Version: "v1", Resource: "clusterroles"},
 		{Group: "rbac.authorization.k8s.io", Version: "v1", Resource: "clusterrolebindings"},
+		sheriffsGVR,
 	}
 
-	for _, gvr := range grantedGVRs {
+	t.Logf("verify that we get empty lists for all claimed resources because the claims have not been accepted yet")
+	for _, gvr := range claimedGVRs {
 		t.Logf("Trying to wildcard list %q", gvr)
 		list, err := dynamicVWClusterClient.Cluster(logicalcluster.Wildcard).Resource(gvr).List(ctx, metav1.ListOptions{})
 		require.NoError(t, err, "error listing %q", gvr)
+		require.Empty(t, list.Items, "expected 0 items but got %#v", list.Items)
+	}
+
+	t.Logf("retrieving cowboys apibinding")
+	apiBinding, err := kcpClusterClient.ApisV1alpha1().APIBindings().Get(logicalcluster.WithCluster(ctx, consumerWorkspace), "cowboys", metav1.GetOptions{})
+	require.NoError(t, err, "error getting cowboys apibinding")
+
+	bindingWithClaims := apiBinding.DeepCopy()
+	for i := range apiBinding.Status.ExportPermissionClaims {
+		claim := apiBinding.Status.ExportPermissionClaims[i]
+		bindingWithClaims.Spec.PermissionClaims = append(bindingWithClaims.Spec.PermissionClaims, apisv1alpha1.AcceptablePermissionClaim{
+			PermissionClaim: claim,
+			State:           apisv1alpha1.ClaimAccepted,
+		})
+	}
+
+	oldJSON := encodeJSON(t, apiBinding)
+	newJSON := encodeJSON(t, bindingWithClaims)
+	patchBytes, err := jsonpatch.CreateMergePatch(oldJSON, newJSON)
+	require.NoError(t, err, "error creating patch")
+	t.Logf("patching apibinding to accept all permissionclaims")
+	apiBinding, err = kcpClusterClient.ApisV1alpha1().APIBindings().Patch(logicalcluster.WithCluster(ctx, consumerWorkspace), apiBinding.Name, types.MergePatchType, patchBytes, metav1.PatchOptions{})
+	require.NoError(t, err, "error patching apibinding to accept all claims")
+
+	t.Logf("waiting for apibinding status to show it's applied the claim for the sheriffs identity")
+	framework.Eventually(t, func() (success bool, reason string) {
+		// Get the binding and make sure that observed permission claims are all set.
+		binding, err := kcpClusterClient.ApisV1alpha1().APIBindings().Get(logicalcluster.WithCluster(ctx, consumerWorkspace), "cowboys", metav1.GetOptions{})
+		if err != nil {
+			return false, err.Error()
+		}
+
+		for _, claim := range binding.Status.AppliedPermissionClaims {
+			if claim.IdentityHash == identityHash {
+				return true, "found applied claim for identity"
+			}
+		}
+		return false, "unable to find applied claim for identity"
+
+	}, wait.ForeverTestTimeout, 100*time.Millisecond, "unable to find applied permission claim for identityHash")
+
+	apifixtures.CreateSheriff(ctx, t, dynamicClusterClient, consumerWorkspace, "wild.wild.west", "in-vw")
```

Comment:
I believe we wanted to deprecate a global fixture, no? cc @sttts

Reply:
It's already deprecated. A follow-up PR should be used to address this.